build/media-suite_compile.sh

```diff
@@ -1357,32 +1359,41 @@ if [[ $xpcomp = "n" && $mpv != "n" ]] && pc_exists libavcodec libavformat libsws
     _check=(EGL/egl.h lib{GLESv2,EGL}.a)
     if ! mpv_disabled egl-angle && ! mpv_disabled egl-angle-lib &&
         do_vcs "https://chromium.googlesource.com/angle/angle" angleproject; then
-        betaversion="$(curl -s http://feeds.feedburner.com/GoogleChromeReleases |\
-            grep -oP "(?<=Beta channel has been updated to )\d+\.\d+\.\d+\.\d+" |\
-            tr '.' ' ' | awk '{print $3}' | head -1)"
-        stablebranch=$(git rev-parse "origin/chromium/$betaversion")
-        if [[ $bits = 64bit ]] && { ! files_exist "${_check[@]}" ||
-            [[ "$(git rev-parse -q --verify stable)" != "$stablebranch" ]]; }; then
-        log stable_checkout git checkout -fB stable "$stablebranch"
-        git clean -qxfd -e "/build_successful*" -e "/recently_updated" -e "*.patch" -e "*.log"
-        do_patch angle-0001-Cross-compile-hacks-for-mpv.patch
-        sed -i "s;'libGLESv2;'libANGLE' ,&;" src/libEGL.gypi
-        sed -i -e '/and OS=="win"/,/}]/d' src/libGLESv2.gypi
-        sed -i -e "s|'copy_compiler_dll.bat', ||" src/angle.gyp
-        sed -r -i "/^\s{8}\[.OS==.win./,$(($(cat src/angle.gyp|wc -l)-2))d" src/angle.gyp
-        log uninstall make PREFIX="$LOCALDESTDIR" \
-            BINDIR="$LOCALDESTDIR/bin-video" uninstall
-        log configure gyp -Duse_ozone=0 -DOS=win \
+        rm -f __ver.txt
+        curl http://omahaproxy.appspot.com/all?os=win64 -o __ver.txt
+        dos2unix -q __ver.txt
+	betaversion="$(sed -rne '/^win64,(stable|beta)/p' __ver.txt | \
+        	sed -re 's;^win64.*,([^,]+),[^,]+$;\1;' | \
+        	grep -E -e '^[3,4][0-9]+$' - | sort -uV | tail -1)"
+		rm -f __ver.txt
```

Comment:
Use pipe if you don't want temp file.
```sh
betaversion="$(curl -s https://omahaproxy.appspot.com/all?os=win64 |\
    dos2unix -q | sed -rne '/^win64,(stable|beta)/p' |\
    tr '.' ' ' | awk '{print $3}' | sort -uV | tail -1 )"
```
Also, you can add a fail-safe version after that.
```sh
betaversion="${betaversion:-3112}"
```